app/src/main/res/values/dimens.xml

```diff
@@ -11,4 +11,7 @@
     <dimen name="dp_5">5dp</dimen>
     <dimen name="dp_160">160dp</dimen>
     <dimen name="sp_17">17sp</dimen>
+    <dimen name="dp200">200dp</dimen>
+    <dimen name="dp30">30dp</dimen>
+    <dimen name="dp15">15dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp15` appears to be unused
 Unused resources make applications larger and slow down builds.